CAKE-5277 | Add CA-specific label and cta

vforge opened the pull request:
https://github.com/Wikia/mobile-wiki/pull/1875

hacks, hacks everywhere.

@Wikia/cake

vforge committed:
CAKE-5277 | Add CA-specific label and cta

mklucsarits commented on extensions/wikia/WatchShow/templates/WatchShow_index.php:
```diff
@@ -1,9 +1,14 @@
+<section class="rail-module watch-show wds-is-hidden" id="watch-show-rail-module-ca" data-tracking-pixel="<?= $trackingPixelURL ?>">
+	<h2><?= $callToActionCA ?></h2>
+	<div class="watch-show__content">
+		<a href="<?= $url ?>" class="wds-button wds-is-secondary" target="_blank" rel="noopener"><?= $buttonLabelCA ?></a>
```
WIll the URL also be different for CA?

vforge replied:
The URL will be the same.

vforge commented:
Added:
- `wgWatchShowButtonLabelMobileCA`
- `wgWatchShowButtonLabelCA`
- `wgWatchShowCTAMobileCA`
- `wgWatchShowCTACA`

Removed:
- Fallbacks for mobile wiki to desktop values if mobile values are not present

vforge committed:
Add more vars and remove mobile fallback
